src/poliastro/tests/tests_twobody/test_orbit.py

```diff
@@ -968,25 +968,30 @@ def test_from_coord_if_coord_is_not_of_shape_zero():
 
 @pytest.mark.remote_data
 def test_from_sbdb():
```

Comment:
Good moment to rename this test? For example

```suggestion
def test_from_sbdb_and_from_horizons_give_similar_results():
```

Reply:
Changed name as suggested